src/Reflection/ReflectionClassConstant.php

```diff
@@ -91,6 +91,16 @@ public function getValue()
     }
 
     /**
+     * Get the value represented as a string.
+     *
+     * @return string
+     */
+    public function getValueAsString() : string
```

Comment:
It could be. But it already was in property reflection (or method parameter... I am not sure).

It is about consistence - the function should be on every reflection where default value is handled or nowhere. Do not you think?